pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project    xmlns="http://maven.apache.org/POM/4.0.0"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>ch-kafka-email</artifactId>
+    <name>ch-kafka-email</name>
+    <version>unversioned</version>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>uk.gov.companieshouse</groupId>
+        <artifactId>companies-house-parent</artifactId>
+        <version>1.0.0-rc2</version>
+    </parent>
+
+    <properties>
+        <!-- JDK -->
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+
+        <!-- Dependency Versions -->
+        <jacoco-maven-plugin.version>0.8.5</jacoco-maven-plugin.version>
```

Comment:
Jacoco Maven plugin in declared in parent pom.xml. I suggest using it to avoid duplications.